awswrangler/distributed/_utils.py

```diff
@@ -1,30 +1,33 @@
 """Utilities Module for Distributed methods."""
 
-from typing import Any, Callable, Dict, List, Optional
+from typing import Any, Callable, Dict, List
 
 import modin.pandas as pd
 import pyarrow as pa
 import ray
 from modin.distributed.dataframe.pandas.partitions import from_partitions
-from ray.data.impl.arrow_block import ArrowBlockAccessor
+from ray.data.impl.arrow_block import ArrowBlockAccessor, ArrowRow
 from ray.data.impl.remote_fn import cached_remote_fn
 
+from awswrangler._arrow import _table_to_df
+
 
 def _block_to_df(
     block: Any,
     kwargs: Dict[str, Any],
-    dtype: Optional[Dict[str, str]] = None,
 ) -> pa.Table:
     block = ArrowBlockAccessor.for_block(block)
-    df = block._table.to_pandas(**kwargs)  # pylint: disable=protected-access
-    return df.astype(dtype=dtype) if dtype else df
```

Comment:
I think this was added to feature-match with non-distributed version. Do you propose to handle this differently or just remove for now?

Reply:
I thought so but then I could not find any other reference in the library. The only one I found was [here](https://github.com/awslabs/aws-data-wrangler/blob/main/awswrangler/dynamodb/_read.py#L97). And even if there was one, I would move it inside this new `_table_to_df` method in order to standardise it

Reply:
Yeah I think this type conversion was done in a different way (probably using map_types when going from pyarrow table to a dataframe), but it wasn't available in distributed case so this was the only crude way to do it

Reply:
Right ok, but do you agree that it's now solved since we are using the same `_table_to_df` method for both the distributed and standard implementations?